experimenter/experimenter/experiments/changelog_utils.py

```diff
@@ -10,10 +10,15 @@
 
 
 class NimbusFeatureConfigChangeLogSerializer(serializers.ModelSerializer):
+    schema = serializers.SerializerMethodField()
+
     class Meta:
         model = NimbusFeatureConfig
         exclude = ("id",)
 
+    def get_schema(self, obj):
+        return obj.schemas.get(version=None).schema
```

Comment:
Can we have the schema's `version` default to `None` so we don't have to explicitly set this everywhere?

Reply:
We can't, django adds the .get method for one-to-many relationships on the field.

Also, a lot of this code that is querying for the `None`-versioned schema will be going away once we actually have versioned schemas in the DB. Right now, all schemas are being imported without a version, but that will change in future patches.